toonz/sources/toonz/main.cpp

```diff
@@ -636,9 +637,14 @@ int main(int argc, char *argv[]) {
   }
 
   QFont *myFont;
-
-  std::string family = EnvSoftwareCurrentFont;
-  myFont             = new QFont(QString(family.c_str()));
+  std::string fontName =
+      Preferences::instance()->getInterfaceFont().toStdString();
+  std::string isBold =
+      Preferences::instance()->getInterfaceFontWeight() ? "Yes" : "No";
+  if (fontName != "") EnvSoftwareCurrentFont = fontName;
+  EnvSoftwareCurrentFontWeight               = isBold;
+  std::string family                         = EnvSoftwareCurrentFont;
```

Comment:
It works fine as it stands.
But I think it would be clear if you could completely remove ENV values `EnvSoftwareCurrentFont` and `EnvSoftwareCurrentFontWeight` as they will not be used anymore.